data/languages/generate_languages.py

```diff
@@ -64,8 +64,10 @@ def get_language_source():
         for key in row:
             # ld[key.strip()] = row[key].strip()
             ld["code"] = row["639-1 "].strip()
-            ld["name"] = row["Language name "].strip()
-            ld["native"] = row["Native name "].strip().strip("\u200e")  # remove left-to-right
+            name = row["Language name "].strip().split(',')[0].split(';')[0] # keep only the first language name if there is a , or ;
+            ld["name"] = name[0].upper() + name[1:] # capitalize first letter and keep the rest as it is (capitalize function lower the rest of the string)
```

Comment:
python has a [capitalize()](https://docs.python.org/3/library/stdtypes.html#str.capitalize) function

Reply:
I tried `capitalize()` at first but it lowers the rest of the string (I wrote it in comments on line 68).
String like ""Bahasa Melayu" would then be converted to ""Bahasa melayu". Another idea is to split, capitalize and concat again.
Anyway the source is not uniform and would require manual editing

Reply:
ok perfect then :)